Source/Core/DolphinQt/Achievements/AchievementHeaderWidget.cpp

```diff
@@ -80,6 +84,24 @@ void AchievementHeaderWidget::UpdateData()
     return;
   }
 
+  const AchievementManager::BadgeStatus& player_badge =
```

Comment:
Do we need to do this if `RA_BADGES_ENABLED` is false?  Maybe move that below and rework the `if` statement into two.